rotkehlchen/data/importer.py

```diff
@@ -297,7 +298,54 @@ def _consume_cryptocom_entry(self, csv_row: Dict[str, Any]) -> None:
                 link='',
             )
             self.db.add_asset_movements([asset_movement])
-
+        elif row_type in ('airdrop_to_exchange_transfer', 'mco_stake_reward'):
+            asset = symbol_to_asset_or_token(csv_row['Currency'])
+            amount = deserialize_asset_amount(csv_row['Amount'])
+            action = LedgerAction(
+                identifier=0,  # whatever is not used at insertion
+                timestamp=timestamp,
+                action_type=LedgerActionType.INCOME,
+                location=Location.CRYPTOCOM,
+                amount=amount,
+                asset=asset,
+                rate=None,
+                rate_asset=None,
+                link=None,
+                notes='',
+            )
+            self.db_ledger.add_ledger_action(action)
+        elif row_type == 'invest_deposit':
+            asset = symbol_to_asset_or_token(csv_row['Currency'])
+            amount = deserialize_asset_amount(csv_row['Amount'])
+            asset_movement = AssetMovement(
+                location=Location.CRYPTOCOM,
+                category=AssetMovementCategory.DEPOSIT,
+                address=None,
+                transaction_id=None,
+                timestamp=timestamp,
+                asset=asset,
+                amount=amount,
+                fee=fee,
+                fee_asset=fee_currency,
+                link='',
```

Comment:
Hmm I just noticed something.

I don't remember why I wrote this comment:

```
    # For exchange asset movements this should be the exchange unique identifier
    # For movements imported from third parties we should generate a unique id for this.
    # If movements are both imported from third parties like cointracking.info and from
    # the exchanges themselves then there is no way to avoid duplicates.
```
https://github.com/rotki/rotki/blob/8443d1ee87f1ebfcb60c7420a992ecae459706a1/rotkehlchen/exchanges/data_structures.py#L48-L52

I am not sure what's the best way to handle this. I guess maybe leaving it empty is fine? The comment refers to the generated `identifier()` which would end up being different if we both import a deposit/withdrawal from an exchange and from something like cointracking.